docker/Dockerfile.ci.dev

```diff
@@ -59,9 +59,9 @@ RUN bash -ex <<"EOF"
         ln -s libnvshmem_host.so.3 libnvshmem_host.so
     popd
 
-    git clone https://github.com/deepseek-ai/DeepEP.git
+    git clone --branch hybrid-ep https://github.com/deepseek-ai/DeepEP.git
```

Comment:
What about also set `version-label` for this `hybrid-ep` branch instead of using commit id which is easier to track and for the following-up version bump-up in the Dockerfile.